### PR TITLE
Update integration guide in README.md

### PR DESCRIPTION
**Pull request by JonBaltz**
Updated the implementation guide in README.md to have more information and ensure all
functions and variables are the most current versions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,59 +36,64 @@ When user back to you page with JWT token, validate it and login user.
...
-      redirectUrl: 'https://your-registered-domain/redirect-path', 
-      cancelUrl: 'https://your-registered-domain/cancel',
-      refId: 'any',
-      scope: 'email'
-    }
-    
-    const requestParams = await silkeySdk.generateSSORequestParams(callerPrivateKey, callerParams)
+1.  Create a ethereum wallet for the program, save the private key
+2.  Save the application in Apollo with the generated wallet, visit https://apollo.silkey.io to do this
+3.  Store the private key in the .env variables for the application
```
Comment (DZariusz) [inline README.md]: lets be more general - there are more ways o store keys in secure way, lets just say  ie: "save key in secure way..."

File: README.md [hunk 2]
```diff
@@ -35,59 +36,64 @@ When user back to you page with JWT token, validate it and login user.
...
+import silkeySdk from "@silkey/sdk";
 
-```
-    const silkeyPublicKey = await fetchSilkeyPublicKey(providerUri, registryAddress)
+// providerUri - A web3 provider URI. ie: 'https://infura.io/v3/:infuraId' register to infura.io to get infuraId
+// registryAddress: Address of silkey smart contract registry, see list of addresses in the registryAddress section of README.md
+const silkeyPublicKey = await silkeySdk.fetchSilkeyPublicKey(providerUri, registryAddress);
 
-    const jwtPayload = silkeySdk.tokenPayloadVerifier(token, silkeyPublicKey)
+const token = queryStringGetter("token");
```
Comment (DZariusz) [inline README.md]: token can be send via POST or GET, lets developers care of getting it. btw. I think there is no form POST autosubmit on athena yet?

File: README.md [hunk 3]
```diff
@@ -35,59 +36,64 @@ When user back to you page with JWT token, validate it and login user.
...
-      // make sure you support both GET and POST redirection,
-      redirectUrl: 'https://your-registered-domain/redirect-path', 
-      cancelUrl: 'https://your-registered-domain/cancel',
-      refId: 'any',
-      scope: 'email'
-    }
-    
-    const requestParams = await silkeySdk.generateSSORequestParams(callerPrivateKey, callerParams)
+1.  Create a ethereum wallet for the program, save the private key
+2.  Save the application in Apollo with the generated wallet, visit https://apollo.silkey.io to do this
```
Comment (DZariusz) [inline README.md]: I would expect more details about this step, maybe some subitems?

File: README.md [hunk 4]
```diff
@@ -35,59 +36,64 @@ When user back to you page with JWT token, validate it and login user.
...
-    const callerParams = {
-      // make sure you support both GET and POST redirection,
-      redirectUrl: 'https://your-registered-domain/redirect-path', 
-      cancelUrl: 'https://your-registered-domain/cancel',
-      refId: 'any',
-      scope: 'email'
-    }
-    
-    const requestParams = await silkeySdk.generateSSORequestParams(callerPrivateKey, callerParams)
+1.  Create a ethereum wallet for the program, save the private key
```
Comment (DZariusz) [inline README.md]: we need to explain that this is for domain, not for a program and how this key will be used

File: README.md (hunk 1 above)
Comment (JonBaltz) [reply README.md]: Okay, Ill change that to be more general

File: README.md (hunk 3 above)
Comment (JonBaltz) [reply README.md]: I'll add more steps about registering the domain using apollo.

File: README.md (hunk 4 above)
Comment (JonBaltz) [reply README.md]: I'll edit the wording to specify that the wallet is linked to the domain when registered with apollo.

**Comment by DZariusz** [comment]
@JonBaltz you dont need to merge develop here... you can remove this PR and branch, I copied your doc instruction into my PR already.

**Pull request merged**